### PR TITLE
Adds custom arcade game text for Halloween and Christmas. (port)

### PR DESCRIPTION
**Pull request by boodaliboo**
original pr https://github.com/tgstation/tgstation/pull/46795
quote from original pr "Adds custom arcade game text for Halloween and Christmas. Now you can EXORCISE THE MACABRE WENDIGO or FREEZE THE YULETIDE KRAMPUS!

Also expands regular arcade game text a bit as well, and moves the arcade game text to json in anticipation of future text variation."
#### Changelog

:cl:  
rscadd: Added various new additions to the arcade 
tweak: tweaked a few text
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
fuck

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -149,19 +149,31 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
...
+		name_part1 = pick_list(ARCADE_FILE, "rpg_adjective_xmas")
+		name_part2 = pick_list(ARCADE_FILE, "rpg_enemy_xmas")
+		weapons = strings(ARCADE_FILE, "rpg_weapon_xmas")
+	else
+		name_action = pick_list(ARCADE_FILE, "rpg_action")
+		name_part1 = pick_list(ARCADE_FILE, "rpg_adjective")
+		name_part2 = pick_list(ARCADE_FILE, "rpg_enemy")
+		weapons = strings(ARCADE_FILE, "rpg_weapon")
+		
+enemy_name = ("The " + name_part1 + " " + name_part2)
```
Comment (JamieD1) [inline code/game/machinery/computer/arcade.dm]: Indent?

Comment (boodaliboo) [reply code/game/machinery/computer/arcade.dm]: ah lets see

Comment (boodaliboo) [reply code/game/machinery/computer/arcade.dm]: nope spacing it still is broken

**Comment by JamieD1** [comment]
Bood, this is like 3 lines, surely you can fix it.

**Comment by boodaliboo** [comment]
i have lost my motivation and i need to take a break this is negatively impacting me

**Pull request closed without merging**